web/src/unread_ui.ts

```diff
@@ -30,11 +30,16 @@ export function update_unread_banner(): void {
 
     const filter = narrow_state.filter();
     const is_conversation_view = filter === undefined ? false : filter.is_conversation_view();
+    toggle_dummy_banner(false);
+
     if (
         user_settings.web_mark_read_on_scroll_policy ===
         web_mark_read_on_scroll_policy_values.never.code
     ) {
         $("#mark_read_on_scroll_state_banner").html(render_mark_as_read_disabled_banner());
+        $("#mark_read_on_scroll_state_banner_place_holder").html(
+            render_mark_as_read_disabled_banner(),
+        );
```

Comment:
Let's extract all these `html` calls into a function which just takes the rendered template as input.

Reply:
Added a prep commit, which extracts them into a separate method.